platform/darwin/src/MGLCircleStyleLayer_Private.h

```diff
@@ -0,0 +1,19 @@
+// This file is generated.
+// Edit platform/darwin/scripts/generate-style-code.js, then run `make darwin-style-code`.
+#import "MGLCircleStyleLayer.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+namespace mbgl {
+    namespace style {
+        class CircleLayer;
+    }
+}
+
+@interface MGLCircleStyleLayer (Private)
+
+- (instancetype)initWithRawLayer:(mbgl::style::CircleLayer *)rawLayer source:(MGLSource *)source;
```

Comment:
Initially, I thought about declaring just `-[MGLSource initWithRawLayer:source:]` (taking `mbgl::style::Layer` in a private header and only declaring specialized methods like this in .m files. But this is safer, so 👍.